geozero-cli/src/main.rs

```diff
@@ -24,7 +27,7 @@ struct Cli {
     #[arg(short, long, value_parser = parse_extent)]
     extent: Option<Extent>,
 
-    /// The path or URL to the FlatGeobuf file to read
+    /// The path or URL to the input file to read
```

Comment:
```suggestion
    /// The path to the input file, or the URL for remote FlatGeobuf files
```

Admittedly the previous phrasing was ambiguous, but it's good to indicate that *only* FGB supports URL inputs.